src/simulation/m_time_steppers.fpp

```diff
@@ -258,6 +268,20 @@ contains
             @:ACC_SETUP_SFs(rhs_vf(i))
         end do
 
+        ! Allocating the cell-average RHS time-stages for adaptive RKCK stepper
+        if (lag_bubbles) then
+            @:ALLOCATE(rhs_ts_adapt(1:6))
```

Comment:
curious if this 1:6 notation works for this data type on Frontier. i think maybe not, not 100% sure though.

Reply:
i don't know why i thought this was an issue. resolving